test/examples/uct_hello_world.c

```diff
@@ -458,6 +486,20 @@ int parse_cmd(int argc, char * const argv[], cmd_args_t *args)
                 return UCS_ERR_UNSUPPORTED;
             }
             break;
+        case 'm':
+            if (!strcmp(optarg, "host")) {
```

Comment:
can we move this parser to common code like ucx_hello_world.h or hello_world_common.c

Reply:
done, moved to ucx_hello_world.h
we don't have .c file for common code for simplicity